vita3k/renderer/include/renderer/gl/functions.h

```diff
@@ -71,7 +71,7 @@ void sync_point_line_width(const GLState &state, const std::uint32_t size, const
 void sync_depth_bias(const int factor, const int unit, const bool front);
 void sync_blending(const GxmRecordState &state, const MemState &mem);
 void sync_texture(GLState &state, GLContext &context, MemState &mem, std::size_t index, SceGxmTexture texture, const Config &config,
-    const std::string &base_path, const std::string &title_id);
+    const std::string &cache_path, const std::string &title_id);
```

Comment:
cache_path is not used now. delete it or replace to log_path and use inside function

Reply:
Fixed.